frontends/election-manager/src/screens/manual_data_import_precinct_screen.tsx

```diff
@@ -380,22 +681,38 @@ export function ManualDataImportPrecinctScreen(): JSX.Element {
               contestTitle = `${contestTitle} - ${party.fullName}`;
             }
           }
+
+          const contestTally = currentPrecinctTally.contestTallies[contest.id];
+          assert(contestTally);
+
           return (
             <ContestData key={contest.id}>
               <Text small>{contest.section}</Text>
               <h3>{contestTitle}</h3>
               <Table borderTop condensed>
                 <tbody>
                   {contest.type === 'candidate' &&
-                    getAllPossibleCandidatesForCandidateContest(contest).map(
+                    getCandidatesFromContestTally(contestTally).map(
```

Comment:
We're now populating this form with candidates from the _results_, which we created to include pre-adjudicated write-ins and could include manual write-ins. Previously we would populate from the candidates in the contest definition.